### PR TITLE
created foodcollect model and cleaned up codes

### PR DESCRIPTION
**Pull request by ZD1wang**
# Description

- created foodcollect model, crud, schemas
- cleaned up the codes
- added foodcollect api tests
- added __init__.py file to the test folder to solve import error

### CODE REVIEW

**Comment by yuedai0531** [comment]
to fix the isort linting error
`isort -rc .` in your CLI

**Comment by yuedai0531** [comment]
how has this been tested?
what are the steps to reproduce the test?

File: backend/tests/api/test_main.py [hunk 1]
```diff
@@ -38,3 +38,32 @@ def test_food(self):
...
+    def test_create_foodCollect(self):
+        payload = {
+            "pickup_time": "2020-12-31T11:11",
+            "vendor_id": 1,
+        }
+        response = self.app.post("/add_foodCollect", json=payload)
+        assert response.status_code == 200
+
+    def test_get_foodCollect(self):
+        response = self.app.post("/get_foodCollect/1")
```
Comment (taerim25) [inline backend/tests/api/test_main.py:51]: should be `self.app.get()`

File: backend/tests/api/test_main.py [hunk 2]
```diff
@@ -38,3 +38,32 @@ def test_food(self):
...
+        assert response.status_code == 200
+
+    def test_get_allFoodCollect(self):
+        response = self.app.post("/get_allFoodCollect")
```
Comment (taerim25) [inline backend/tests/api/test_main.py:55]: `self.app.get()`

File: backend/tests/api/test_main.py [hunk 3]
```diff
@@ -38,3 +38,32 @@ def test_food(self):
...
+        assert response.status_code == 200
+
+    def test_update_foodCollect(self):
+        payload = {
+            "pickup_time": "2030-04-01T11:11",
+            "vendor_id": 1,
+            "id": 1,
+        }
+        response = self.app.post("/update_foodCollect", json=payload)
```
Comment (taerim25) [inline backend/tests/api/test_main.py:64]: Should be `self.app.put()`

File: backend/app/main/crud.py [hunk 4]
```diff
@@ -43,3 +47,41 @@ def remove_food(session: Session, food_id: int):
     session.query(models.Food).filter(models.Food.id == food_id).delete()
     session.commit()
     return food_id
+
+
+def create_foodCollect(session: Session, foodCollect: schemas.FoodCollectCreate):
+    db_foodCollect = models.FoodCollect(
+        pickup_time=foodCollect.pickup_time, vendor_id=foodCollect.vendor_id,
```
Comment (taerim25) [inline backend/app/main/crud.py:54]: Some parameters maybe missing in this call.

**Comment by ZD1wang** [comment]
https://stackoverflow.com/questions/56780892/python-unit-test-module-throws-modulenotfounderror-no-module-named-tests-test

If you remove the __init__.py file, Python will no longer look for submodules inside that directory, so attempts to import the module will fail.

File: backend/app/main/crud.py [hunk 5]
```diff
@@ -65,3 +69,41 @@ def remove_tray(session: Session, tray_id: int):
...
+    )
+    session.add(db_foodCollect)
+    session.commit()
+    session.refresh(db_foodCollect)
+    return db_foodCollect
+
+
+def remove_foodCollect(session: Session, foodCollect_id: int):
```
[the remark this answers is not in the document]
Comment (Ty-Won) [reply backend/app/main/crud.py:84]: good catch, I noticed there is some use of _ and other camel case. Best to decide on 1 convention

[the remark this answers is not in the document]
Comment (ZD1wang) [reply backend/app/main/crud.py]: > good catch, I noticed there is some use of _ and other camel case. Best to decide on 1 convention

Please check the changes :)

**Comment by Ty-Won** [comment]
@ZD1wang Just a quick reminder as well, once you merge this, be sure to see how it affects your other PR ;o I think there is some overlap between work done there and here such as getAllFoodCollect

**Pull request merged**